nbs/src/tbats.ipynb

```diff
@@ -26,11 +26,11 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.            for i in range(1, h+1):
</pre><p>I think we're recomputing these several times. We could probably allocate an array upfront, fill it and then slice it in every loop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>this was a great suggestion, so I implemented it. </p>